test/e2e/build_test.go

```diff
@@ -438,36 +450,22 @@ RUN find /test`, CITEST_IMAGE)
 		Expect(session.OutputToString()).To(ContainSubstring("/test/dummy"))
 	})
 
-	It("podman remote build must not allow symlink for ignore files", func() {
-		// Create a random file where symlink must be resolved
-		// but build should not be able to access it.
-		privateFile := filepath.Join("/tmp", "private_file")
+	It("podman build with .dockerignore symlink pointing outside context dir", func() {
+		// dockerignore file outside the context directory
+		privateFile := filepath.Join(podmanTest.TempDir, "private_file")
 		f, err := os.Create(privateFile)
 		Expect(err).ToNot(HaveOccurred())
-		// Mark hello to be ignored in outerfile, but it should not be ignored.
 		_, err = f.WriteString("hello\n")
 		Expect(err).ToNot(HaveOccurred())
-		defer f.Close()
+		f.Close()
 
-		// Create .dockerignore which is a symlink to /tmp/private_file.
-		currentDir, err := os.Getwd()
-		Expect(err).ToNot(HaveOccurred())
-		ignoreFile := filepath.Join(currentDir, "build/containerignore-symlink/.dockerignore")
+		// Link to .dockerignore outside context dir
+		contextDir := tempContextDirectory(podmanTest, "containerignore-symlink")
+		ignoreFile := filepath.Join(contextDir, ".dockerignore")
 		err = os.Symlink(privateFile, ignoreFile)
 		Expect(err).ToNot(HaveOccurred())
-		// Remove created .dockerignore for this test when test ends.
-		defer func() {
-			os.Remove(ignoreFile)
-		}()
-
-		if IsRemote() {
-			podmanTest.StopRemoteService()
-			podmanTest.StartRemoteService()
```

Comment:
Judgment call: I have no idea why this was necessary, and I deem it pointless. Nuked.